mesonbuild/modules/pkgconfig.py

```diff
@@ -271,8 +271,7 @@ def add_version_reqs(self, name: str, version_reqs: T.Optional[T.List[str]]) ->
             # Note that pkg-config is picky about whitespace.
             # 'foo > 1.2' is ok but 'foo>1.2' is not.
             # foo, bar' is ok, but 'foo,bar' is not.
-            new_vreqs = [s for s in mesonlib.stringlistify(version_reqs)]
-            self.version_reqs[name].update(new_vreqs)
+            self.version_reqs[name].update(version_reqs)
```

Comment:
This code is weird because it seems like it assumed that version_reqs could be something other than None or List[str], so it attempted to redundantly stringlistify it... fascinating...

LGTM